services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/northbound/switches/MetersSpec.groovy

```diff
@@ -0,0 +1,261 @@
+    }
+
+    def "Default meters should express bandwidth in packet/s rather than kbps on the non-Centec switches"() {
+        given: "All Openflow 1.3 compatible switches"
+        this.requireProfiles("hardware") //TODO: Research how this behaves on OpenVSwitch
+        def switches = getNonCentecSwitches()
+
+        when: "Try to get meters from all the switches"
+        List<SwitchMeterEntries> meters = switches.collect {
+            northboundService.getAllMeters(it.dpId)
+        }
+
+        then: "Only the default meters should be present on each switch"
+        meters.each {
+            assert it.meterEntries.size() == 2
+            assert it.meterEntries.every { it.rate == DISCO_PKT_RATE }
+            assert it.meterEntries.every {
+                //NoviFlow wouldn't let us set burst size to more than 105% of the flow bandwidth
+                //effectively negating the default burst packet rate
+                DISCO_PKT_RATE >= 4096 ? it.burstSize == DISCO_PKT_BURST :
+                        (it.burstSize >= Math.floor(DISCO_PKT_RATE * NOVI_BURST_MIN)) &&
+                                (it.burstSize <= Math.ceil(DISCO_PKT_RATE * NOVI_BURST_MAX))
+            }
+            assert it.meterEntries.every { it.meterId <= Constants.MAX_DEFAULT_METER_ID }
+            assert it.meterEntries.every { ["PKTPS", "BURST", "STATS"].containsAll(it.flags) }
+            assert it.meterEntries.every { it.flags.size() == 3 }
+        }
+    }
+
+    def "Creating a flow should create new meters on a switch"() {
+        given: "A switch with OpenFlow 1.3 support"
+        def sw = topology.getActiveSwitches().find { it.ofVersion == "OF_13" }
+
+        when: "Get default meters from the switch"
+        def defaultMeters = northboundService.getAllMeters(sw.dpId)
+        assert defaultMeters
+
+        and: "Create a flow"
+        def flow = flowHelper.addFlow(flowHelper.singleSwitchFlow(sw))
+
+        then: "New meters should appear after flow setup"
+        def newMeters = northboundService.getAllMeters(sw.dpId)
+        def newMeterEntries = newMeters.meterEntries.findAll { !defaultMeters.meterEntries.contains(it) }
+        newMeterEntries.size() == 2
+
+        and: "All new meters should have KBPS, BURST and STATS flags installed"
+        newMeterEntries.every { it.flags.sort().equals(["KBPS", "BURST", "STATS"].sort()) }
+
+        and: "All new meters rate should be equal to flow's rate"
+        newMeterEntries*.rate.every { it == flow.maximumBandwidth }
+
+        when: "Delete a flow"
+        flowHelper.deleteFlow(flow.id)
+
+        then: "New meters should disappear from the switch"
+        def newestMeters = northboundService.getAllMeters(sw.dpId)
+        newestMeters.meterEntries.containsAll(defaultMeters.meterEntries)
+        newestMeters.meterEntries.size() == defaultMeters.meterEntries.size()
+    }
+
+    @Unroll
+    def "Meter burst size should not exceed 105% of #flowRate kbps on non-Centec switches"() {
```

Comment:
105% is configurable, isn't it? So why shouldn't we pick this value from config instead of hardcoding?